test/commands-tests/history.js

```diff
@@ -204,4 +230,82 @@ describe('ForgAE History', async () => {
             process.chdir(currentCwd);
         });
     });
+
+    describe.only('History', async () => {
```

Comment:
What is the difference between this "describe" and the first one? if there is you can change the describe name

Reply:
The difference is that I use `beforeEach` not `before`